src/docs/content/observability/oci-monitoring.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "OCI Monitoring"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 90
+tags:
+  - Monitoring
+  - Metrics
+---
+
+{{% alert style="warning" icon="warning" %}}
+Note that this is **OPTIONAL**.  
+{{% /alert %}}
+
+## Introduction
+
+The Oracle Cloud Infrastructure Monitoring service enables you to actively and passively monitor your cloud resources using the Metrics and Alarms features. Read more details [here](https://docs.cloud.oracle.com/en-us/iaas/Content/Monitoring/Concepts/monitoringoverview.htm)
```

Comment:
... enables active and passive monitoring of cloud resources... would be better use of OCI voice guidelines